README.md

```diff
@@ -0,0 +1,24 @@
+# ros2cli
+
+This repository contains the source code for ROS 2 command line interface tools included with a standard install of any ROS 2 distro. 
+
+## Usage
+
+Run `ros2 --help` to see all available verbs.
+
+Run `ros2 <verb> --help` for more information on individual command usage.
+
+Read [Introspection with command line tools](https://index.ros.org/doc/ros2/Tutorials/Introspection-with-command-line-tools/) on ROS Index for more information and an example.
+
+### Cheat Sheet
+
+This [cheat sheet](https://github.com/artivis/ros2_cheats_sheet/blob/master/cli/cli_cheats_sheet.pdf) provides examples for verbs and their sub-verbs (some of which rely on the [ROS 2 demos package](https://github.com/ros2/demos)).
```

Comment:
I think we use the words `command` and `verb` (and not `verb` and `subverb`) throughout the code. I don't know what the correct words are (the cheatsheet also use verb and subverb).

Reply:
I also don't know what the correct words are, so I would say pick something and be consistent with it?

Reply:
Let me know about the appropriate wording so that I can update the cheats sheet if necessary :+1:

Reply:
I think that using `command` and `verb` is more consistent with the existing code.

Reply:
`ros2 -h` uses `<command>` for both the first and the second argument, e.g. when I run `ros2 bag -h` it says `call 'ros2 bag <command> -h' for more detailed usage`. Any thoughts about that?

Reply:
I just noticed it also says `Various rosbag related sub-commands`. So it seems like the code says uses different terminology in different places